README.md

```diff
@@ -40,16 +40,21 @@ composer require --dev rector/rector 'dev-master' nikic/php-parser '4.0.x-dev'
 
 ### Do you Have Conflicts?
 
-It may also happen your dependencies are in conflict, e.g. PHPStan requires `nikic/php-parser 3.0`, or older PHPUnit etc. This might be solved in the future, when `nikic/php-parser` 4 becomes stable.
+It may also happen your dependencies are in conflict, e.g. PHPStan requires PHP-Parser version 3, or older PHPUnit etc. This might be solved in the future, when PHP-Parser version 4 becomes stable.
```

Comment:
Why this change?

Reply:
I think to write `PHP-Parser` instead of `nikic/php-parser` is better, as we are explaining something, not require or so :smile:

Reply:
As this is less popular package, it might be harder to find. Composer package name is fqn for it.

But if you insist, at least add link to the GitHub repository for it.

Reply:
Let's also link PHPStan? As isn't so popular as well

Reply:
Yes, thanks